<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>58</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetEmergency">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEmergency">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDoors">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open doors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCloseDoors">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Doors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS LED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GO_DOWN">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Go down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_go_UP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Go up</string>
    </property>
   </widget>
   <widget class="QLabel" name="DOWN_LED">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWNLED</string>
    </property>
   </widget>
   <widget class="QLabel" name="UP_LED">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>UPLED</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
